jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/GraphOrdering.java

```diff
@@ -161,21 +172,21 @@ public int getVertexCount()
      */
     public boolean hasEdge(int v1Number, int v2Number)
     {
-        V v1, v2;
-        Boolean containsEdge = null;
-
+        
+        final int cacheIndex = 2*(v1Number*vertexCount+v2Number);
```

Comment:
Initialize to 0 and defer computation.  (Probably the compiler is smart enough to do this anyway, but hey.

Reply:
yeah, I'm rather sure the JIT would have even figured out the double cacheIndex computation before. but better be safe than sorry.

done.